test/crossfilter.spec.js

```diff
@@ -871,14 +871,14 @@ describe("crossfilter", () => {
       it("can use a samplingRatio < 1 with filterQuery", () => {
         dimension.samplingRatio(0.5)
         expect(dimension.writeTopQuery(Infinity)).to.eq(
-          "SELECT users.id FROM users WHERE  MOD(users.rowid * 265445761, 4294967296) < 2147483648 ORDER BY users.id DESC"
+          "SELECT users.id FROM users WHERE  MOD(users.rowid * 2654435761, 4294967296) < 2147483648 ORDER BY users.id DESC"
         ) // TODO extra space between WHERE & MOD
       })
       it("can use a samplingRatio < 1 without filterQuery", () => {
         dimension.filterExact(1)
         dimension.samplingRatio(0.5)
         expect(dimension.writeTopQuery(Infinity)).to.eq(
-          "SELECT users.id FROM users WHERE users.id = 1 AND  MOD(users.rowid * 265445761, 4294967296) < 2147483648 ORDER BY users.id DESC"
+          "SELECT users.id FROM users WHERE users.id = 1 AND  MOD(users.rowid * 2654435761, 4294967296) < 2147483648 ORDER BY users.id DESC"
```

Comment:
If these queries are actually sampling queries, we should use expanded modulo arithmetic here as well so distributed environment doesn't throw a false exception.
Arithmetic change we need to do is `MOD(A * K, B)`   => `MOD( MOD(A , B) * MOD(K , B), B)`